src/components/TokenForm.js

```diff
@@ -35,15 +35,14 @@ const TokenForm = ({ setToken }) => {
               value={tokenInput}
               onChange={(event) => {
                 setTokenInput(event.target.value);
+                setError(false);
```

Comment:
Took suggestion from Chris to set the error state to false so that the error goes away while a user is typing